tests/functional/m/misplaced_comparison_constant.py

```diff
@@ -34,3 +34,14 @@ def good_comparison():
     for i in range(10):
         if i == 5:
             pass
+
+def double_comparison():
+    """Check that we return early for non-binary comparison"""
+    for i in range(10):
+        if i == 1 == 2:
+            pass
```

Comment:
```suggestion
        if i == 1 == 2:
            pass
        if 2 <= i <= 8:
            print("Between 2 and 8 inclusive")
```
I think adding the standard use case could be safe as the other one is kinda theoretical.